src/Simplic.Package.Service/ExtensionHelper.cs

```diff
@@ -7,10 +11,107 @@ namespace Simplic.Package.Service
     /// </summary>
     public static class ExtensionHelper
     {
+        private static string tempPath;
+
         /// <summary>
         /// Gets a list of all loaded extension names.
         /// </summary>
         public static IList<string> LoadedExtensions = new List<string>();
 
+        /// <summary>
+        /// Gets a temp path for extensions.
+        /// <para>
+        /// This will create a new path if none exists and will add it to the app domain.
+        /// </para>
+        /// </summary>
+        /// <returns>The path</returns>
+        public static string GetTempPath()
+        {
+            if (!string.IsNullOrWhiteSpace(tempPath))
+                return tempPath;
+
+            tempPath = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
+
+            if (!Directory.Exists(tempPath))
```

Comment:
AFAIK you don't need this line. CreateDirectory will check this. But it is ok.

Reply:
I,ve checked the documentation, you are right